figure.html

```diff
@@ -398,6 +398,131 @@
+</div> <!-- end #line-block -->
+
+<img alt="../images/figureROICopy.jpg" class="align-center" src="images/figureROICopy.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<li>
+<p class="first">Rectangular ROIs on one panel can be used to crop other panels to the same area.</p>
+
+<p>Select the panel with the ROI on it.</p>
+
+<p>Click <span class="bold">Copy</span> in the ROIs toolbar in the <span class="bold">Labels</span> tab.</p>
+
+<p>Select the panel to be cropped.</p>
+
+<p>In the <span class="bold">Preview</span> tab, click <span class="bold">Paste</span>.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/figurePasteCrop.jpg" class="align-center" src="images/figurePasteCrop.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<li>
+<p class="first">Non-microscopy images such as scans of Western blots or exported charts can be imported to the OMERO server in TIFF, PNG or other format and used in Figure.</p>
+
+<p>Select the required images in OMERO.web, copy the link and add to the Figure canvas as before.</p>
+
+<p>These images can then be resized, zoomed, cropped and ROIs added as described.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/figureBlot.jpg" class="align-center" src="images/figureBlot.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<li>
 <p class="first">Click <span class="bold">Edit Figure Legend</span> to add a legend to selected images.</p>
```

Comment:
I think you can only add a legend to the whole Figure as of now, not specifically to selected images, @will-moore ?